sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Constants.cs

```diff
@@ -24,5 +24,8 @@ internal static class Constants
     {
         internal const string NuGetLatestVersion = "LATEST";
         internal const string DotNetCore220Version = "2.2.0";
+
+        internal const string EmptyNullableValueAccess = "S3655";
```

Comment:
I think these can stay just inside `SymbolicExecutionAnalyzerFactoryTest.cs`, no need to make them internal

Reply:
Indeed. We can extract them later if needed.